### PR TITLE
Add PAA 7% OT information

### PR DESCRIPTION
**Pull request by kevinkiklee**
This PR adds PAA 7% OT ramp-up information on the OT status page. 

Preview link: https://pr-6496-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/unified-origin-trial/#june-2023

### CODE REVIEW

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: >
    Run unified experiments across Attribution Reporting, Protected Audience API,
    Topics, Fenced Frames, and Shared Storage.
 date: 2022-09-08
-updated: 2023-03-13
+updated: 2023-06-02
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:11]:
```suggestion
updated: 2023-06-05
```

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 2]
```diff
@@ -29,11 +29,119 @@ across the Topics, Protected Audience API, and Attribution Reporting APIs. This
 through the configuration steps to access the APIs, tells you how to validate
 your configuration, and provides further resources for testing against the APIs.
 
+{% Aside %}
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:32]:
```suggestion
{% Aside 'key-term' %}
```

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 3]
```diff
@@ -29,11 +29,119 @@ across the Topics, Protected Audience API, and Attribution Reporting APIs. This
...
+Note: The term "traffic" is used to refer to the percentage of Chrome browser instances included in the trial, not the percentage of requests made to individual sites.
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:33]:
```suggestion
_Traffic_ refers to the percentage of Chrome browser instances included in the trial, not the percentage of requests made to individual sites.
```

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 4]
```diff
@@ -122,19 +230,19 @@ The traffic allocation will look like the following starting on Monday, April 17
   </tr>
 </table>
 
-### March 2023
+#### March 2023
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:233]: The January entry also needs to have an extra `#` added.

**Review by rowan-m — APPROVED** [review]
LGTM

**Review by mihajlija — APPROVED** [review]

**Pull request merged**